corehq/apps/reports/filters/users.py

```diff
@@ -340,7 +340,7 @@ def filter_context(self):
     @classmethod
     def user_es_query(cls, domain, mobile_user_and_group_slugs, request_user):
         # The queryset returned by this method is location-safe
-        q = user_es.UserES().domain(domain)
+        q = user_es.UserES().domain(domain, allow_mirroring=True)
```

Comment:
Not knowing how widespread this function is, are there any concerns this could cause unexpected users to show up somewhere we don't want them? E.g. in a dropdown or something...

Reply:
Good question; I had gone through `UserES` usages but not all `user_es_query` usages. Doing that now, the current behavior seems right.

This is used in a bunch of standard monitoring & deployment reports and when generating the enterprise dashboard form submissions report, all places where projects should see activity done by mirrored users. The mirrored users don't show up in the user dropdowns - I didn't look into what in the code controls them, just tested on staging.